core/print_api.php

```diff
@@ -1248,10 +1248,6 @@ function print_custom_field_projects_list( $p_field_id ) {
  * @return void
  */
 function print_plugin_priority_list( $p_priority ) {
-	if( $p_priority < 1 && $p_priority > 5 ) {
-		echo '<option value="', $p_priority, '" selected="selected">', $p_priority, '</option>';
-	}
-
```

Comment:
I wouldn't qualify this as useless. Even though it's unlikely to happen as there is no path for that in Mantis core, the priority *could* technically have a value other than 1..5; this code ensures that whatever is stored in the database does not get silently overwritten when the form is submitted.

I recommend to not merge this commit.

Reply:
Yes you're right, I didn't check the actual condition :blush:... Should be `||` then.